src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/feil/FeilService.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.sbl.sosialhjelp_mock_alt.datastore.feil
+
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import no.nav.sbl.sosialhjelp_mock_alt.KonfigurertFeil
+import no.nav.sbl.sosialhjelp_mock_alt.utils.hentFnrFraToken
+import no.nav.sbl.sosialhjelp_mock_alt.utils.randomInt
+import org.springframework.http.HttpHeaders
+import org.springframework.stereotype.Service
+
+@Service
+class FeilService {
+    private val feilsituasjoner = HashMap<String, Feilsituasjon>()
+
+    fun legtilFeil(
```

Comment:
skriveleif? `leg` --> `legg` 

Jeg er generelt opplært til å la domene-objekter være på norsk (fordi det er ord som er spesifikt for dette prosjektet) 
Men å la typiske programmeringsord være på engelsk. "getFeil", "addFeil", "createFeil", "isFeilPresent" etc. 
Jeg tenker dette er noe vi kan diskutere på utviklerpraten.

Reply:
Jeg retta opp skrivefeilen, og er klar for litt vennlig krangel ;-)